addons/batikvis/src/main/java/de/lmu/ifi/dbs/elki/visualization/gui/SelectionTableWindow.java

```diff
@@ -290,10 +290,7 @@ public String getColumnName(int column) {
 
     @Override
     public boolean isCellEditable(int rowIndex, int columnIndex) {
-      if(columnIndex == 0) {
-        return false;
-      }
-      return true;
+      return columnIndex != 0;
```

Comment:
I indeed did not check that. If that's the case, feel free to ignore this change.